WebDriverAgentLib/Utilities/FBConfiguration.h

```diff
@@ -155,6 +166,13 @@ NS_ASSUME_NONNULL_BEGIN
  * @param key The key name to set @c value to
  */
 - (void)setValue:(NSValue *)value forPreferenceKey:(NSString *)key;
+
+/**
+ * Get the preferenve by @c key
+ *
+ * @param key The key name to get the value
+ */
+- (BOOL)boolForPreferenceKey:(NSString *)key;
```

Comment:
I'd rather return the id type to unify raw the method

Reply:
or NSValue

Reply:
here is a private header, so I moved to `PrivateHeaders` dir